build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.reaktive_version = '1.1.22'
+    ext.reaktive_version = '1.1.23-t1'
```

Comment:
Why `-t1`?

Reply:
Accidentally, was checking the publication, thanks for spotting!